src/App.test.jsx

```diff
@@ -1,8 +1,23 @@
+
+/* eslint-disable import/extensions */
 import React from 'react';
-import ReactDOM from 'react-dom';
-import App from './App';
+import toJson from 'enzyme-to-json';
+import { shallow } from 'enzyme';
+import configureStore from 'redux-mock-store';
+import thunkMiddleware from 'redux-thunk';
+import { Provider } from 'react-redux';
+import SHEET_API from './utils/Api';
+import App from './App.jsx';
 
-it('renders without crashing', () => {
-  const div = document.createElement('div');
-  ReactDOM.render(<App />, div);
+const middleware = [ thunkMiddleware.withExtraArgument({ SHEET_API })];
+const mockStore = configureStore(middleware);
+describe('App component', () => {
+  it('should render a component with props as specified ', () => {
```

Comment:
@Gwillison415 perhaps the `connected` component was used in order to write this test?

@HamidAghdaee would you mind weighing in on this one here? Thanks!

Reply:
yeah I agree with @joshwyatt Don't test the connected component, export the unconnected one and test it.